server/codies.js

```diff
@@ -54,41 +67,68 @@ export default class Codies {
 
   constructor() {
     this.redTeam = [];
-    this.blue_team = [];
+    this.blueTeam = [];
     this.generateBoard();
   }
 
-  addRedPlayer(player_name) {
-    this.redTeam.push(player_name);
-  }
-
-  addBluePlayer(player_name) {
-    this.blue_team.push(player_name);
-  }
-
-  removeRedPlayer(player_name) {
-    const index = this.redTeam.indexOf(player_name);
-    if (index > -1) {
-      this.redTeam.splice(index, 1);
+  addPlayer(playerName, playerId) {
+    if (this.redTeam.length > this.blueTeam.length) {
+        this.blueTeam.push(new Player(playerName, playerId));
+    } else {
+        this.redTeam.push(new Player(playerName, playerId));
     }
   }
 
-  removeBluePlayer(player_name) {
-    const index = this.blue_team.indexOf(player_name);
+  removePlayer(playerId) {
+    const index = findPlayer(playerId, this.redTeam);
     if (index > -1) {
-      this.blue_team.splice(index, 1);
+      this.redTeam.splice(index, 1);
+    } else {
+      const index = findPlayer(playerId, this.blueTeam);
+      if (index > -1) {
+        this.blueTeam.splice(index, 1);
+      }
```

Comment:
did I ever end up adding lodash? lodash can make things like this really easy and more readable.

```
// removes item from array that matches a function or object properties
_.remove(redTeam, { playerId });
```

Reply:
I think you did? Will change